content/influxdb/v2.0/reference/glossary.md

```diff
@@ -259,9 +259,24 @@ Related entries: [bucket](#bucket)
 
 ### data type
 
-A data type is defined by the values it can take, the programming language used, or the operations that can be performed on it.
-
-InfluxDB supports the following data types: float, integer, string, boolean, and timestamp.
+A data type is defined by the values it can take, the programming language used, or the operations that can be performed on it. 
+
+InfluxDB supports the following data types: 
+- boolean 
+- dateTime
+- double 
+- field
+- float
+- ignored
+- integer
+- long 
+- measurement 
+- string
+- tag
+- timestamp 
+- unsignedLong 
+
+For more information about each data type, see [extended annotated CSV](/influxdb/cloud/reference/syntax/annotated-csv/extended/#datatype). 
```

Comment:
This is probably worthy of a separate PR, but I think we should add a Data types reference doc that captures the content from [Line protocol – Data types and format](https://docs.influxdata.com/influxdb/v2.0/reference/syntax/line-protocol/#data-types-and-format) as well as the [Annotated CSV docs](https://docs.influxdata.com/influxdb/v2.0/reference/syntax/annotated-csv/) in one place and then reference that doc here.

Thoughts @kelseiv ☝🏻 ?

Reply:
Sounds good